pycode/memilio-epidata/memilio/epidata_test/test_epidata_get_population_data.py

```diff
@@ -134,8 +134,10 @@ def test_get_population(self, mock_data):
             read_data=True, file_format='json', out_folder=self.path,
             no_raw=False, split_gender=False, merge_eisenach=False)
 
+        # add seven to the number of test_counties as this is the current workaround to add counties which are
+        # not mentioned in old data sets
         test_df = pd.read_json(os.path.join(
-            self.path, 'Germany/', 'county_current_population_dim401.json'))
+            self.path, 'Germany/', 'county_current_population_dim' + str(len(self.test_counties)+7) +'.json'))
```

Comment:
I think the function `get_new_counties` should not add those 7 counties if there is no data for them.
The workaround should be okay for now, but maybe we should open a new Issue or at least a TODO.

Reply:
See https://github.com/DLR-SC/memilio/issues/405